internal/campaigns/types.go

```diff
@@ -1456,3 +1460,70 @@ const (
 func unixMilliToTime(ms int64) time.Time {
 	return time.Unix(0, ms*int64(time.Millisecond))
 }
+
+type Action struct {
+	ID             int64
+	CampaignID     *int64
```

Comment:
```suggestion
	CampaignID     int64
```

We use `0` to mean "not set"